olad/www/new/views/universe-add.html

```diff
@@ -5,7 +5,7 @@
  </div>
  <div class="form-group">
   <label for="UniverseName" class="control-label">Name:</label>
-  <input placeholder="downstage-spots" required id="UniverseName" class="form-control" type="text" ng-model="Data.name"/>
+  <input placeholder="dimmer" required id="UniverseName" class="form-control" type="text" ng-model="Data.name"/>
```

Comment:
Can we call it dimmers as the example text please, rather than the singular form.